src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
+
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    static final String White_Space = " ";
+    static final int Date_Index = 0;
+    static final int Name_Index = 1;
+    static final int Hours_Index = 2;
+    static final int Salary_Index = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        LocalDate currentDateFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate currentDateTo = LocalDate.parse(dateTo, formatter);
+        StringBuilder report = new StringBuilder("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
```

Comment:
It can be a constant
```suggestion
                .append(SEPARATOR)
```